test.py

```diff
@@ -177,14 +191,6 @@ def test(bad, good):
         yield test, bad, good
 
 
-def test_strict_unicode_escape():
```

Comment:
`URL`s can no longer be constructed component-wise like this, and are converted from unicode to UTF-8 at `parse` time.